src/common/crash_dump/crash_dump_distribute.hpp

```diff
@@ -13,6 +13,8 @@ inline constexpr uint16_t port { 8888 };                  // temporary port
 
 inline constexpr size_t url_buff_size { 128 };
 
+inline constexpr uint8_t socket_timeout_s { 6 };
```

Comment:
Shall these constants be publicly available in the header file (both of them), or are they more of an implementation fine-tuning parameters the caller is not interested in?

Reply:
The reason I've put them in the header is so that 'they're easy to find'.. but I see your point and I will change it accordingly, it makes sense